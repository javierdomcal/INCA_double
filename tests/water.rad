  0.0000    0.1982753843007469D+02
  0.0500    0.1827655318218336D+02
  0.1000    0.1467791824155556D+02
  0.1500    0.1087233536771619D+02
  0.2000    0.7843549014563868D+01
  0.2500    0.5732174012670445D+01
  0.3000    0.4380257009385553D+01
  0.3500    0.3574739373583034D+01
  0.4000    0.3117657595656656D+01
  0.4500    0.2857610201742219D+01
  0.5000    0.2695994440159824D+01
  0.5500    0.2576399323145959D+01
  0.6000    0.2470157741645397D+01
  0.6500    0.2364679770141391D+01
  0.7000    0.2255782949811701D+01
  0.7500    0.2143214139990401D+01
  0.8000    0.2028258453323183D+01
  0.8500    0.1912572962201317D+01
  0.9000    0.1797688772430379D+01
  0.9500    0.1684853608725712D+01
  1.0000    0.1575029969270487D+01
  1.0500    0.1468948293436865D+01
  1.1000    0.1367164724826344D+01
  1.1500    0.1270103083656496D+01
  1.2000    0.1178077757489241D+01
  1.2500    0.1091302439205614D+01
  1.3000    0.1009891990144012D+01
  1.3500    0.9338637032009198D+00
  1.4000    0.8631418629161158D+00
  1.4500    0.7975670627656355D+00
  1.5000    0.7369099169445819D+00
  1.5500    0.6808877745120711D+00
  1.6000    0.6291826941259944D+00
  1.6500    0.5814590389152747D+00
  1.7000    0.5373793800513256D+00
  1.7500    0.4966177947481963D+00
  1.8000    0.4588700239597751D+00
  1.8500    0.4238602871557176D+00
  1.9000    0.3913448353794579D+00
  1.9500    0.3611125640443880D+00
  2.0000    0.3329832008933533D+00
  2.0000    0.3329832008933533D+00
  2.1000    0.2824436976123623D+00
  2.2000    0.2387434983904584D+00
  2.3000    0.2011099580721816D+00
  2.4000    0.1688668551496267D+00
  2.5000    0.1413781796114656D+00
  2.6000    0.1180383684082153D+00
  2.7000    0.9828438007185691D-01
  2.8000    0.8160941788503291D-01
  2.9000    0.6756910211120944D-01
  3.0000    0.5577956962726400D-01
  3.1000    0.4591074573944494D-01
  3.2000    0.3767822778408727D-01
  3.3000    0.3083594210167633D-01
  3.4000    0.2517037490973758D-01
  3.5000    0.2049629522437240D-01
  3.6000    0.1665352828465446D-01
  3.7000    0.1350432701604929D-01
  3.8000    0.1093102861300108D-01
  3.9000    0.8833838846554828D-02
  4.0000    0.7128698897962991D-02
  4.1000    0.5745249339949632D-02
  4.2000    0.4624926136594957D-02
  4.3000    0.3719220892402004D-02
  4.4000    0.2988125181043444D-02
  4.5000    0.2398764791656090D-02
  4.6000    0.1924218234724599D-02
  4.7000    0.1542506206825833D-02
  4.8000    0.1235734724024827D-02
  4.9000    0.9893734102754885D-03
  5.0000    0.7916509554322473D-03
  5.1000    0.6330511925532359D-03
  5.2000    0.5058950115551443D-03
  5.3000    0.4039951034304427D-03
  5.4000    0.3223721751662393D-03
  5.5000    0.2570227544642884D-03
  5.6000    0.2047300260982899D-03
  5.7000    0.1629103301806482D-03
  5.8000    0.1294890240666823D-03
  5.9000    0.1028003733660456D-03
  6.0000    0.8150699708386620D-04
