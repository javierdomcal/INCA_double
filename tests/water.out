 *******Job Finished************
 -----------Computational time----------------
 Total CPU time   39.7442131    
 Grid points computing time   4.07100022E-02
 Primitive quartet loop time   39.7029877    
 CPU time for intracule integrations   5.14984131E-04
 ---Primitive quartet time analysis-----------
 Reading .dm2p file-->   3.87137085E-02
 1st primitive screening-->  -7.92128444E-02
 2nd primitive screening-->  0.144801080    
 Grid points loop-->   38.6586723    
 ---------------------------------------------
 ------Grid point + primitive quartet info----
 Original grid points           0
 Symmetry reduced grid points        5051
 Total number of reduced grid points           0
 ---------------------------------------------
 --------------Accuracy check-----------------
 Sum of all DM2prim terms=   33.703058128264622        23693.806767268470     
 Thresholds used for DM2prim   10.000000000000000        10.000000000000000     
 Threshold used for screenings(Tau)=   1.0000000000000000E-008
 Computed limit for the screenings   4.3290043290043294E-011
 ---------------------------------------------
 RADIAL PLOT, I(S) vs s
 Number of distances          82
 From   0.0000000000000000      to   6.0000000000000000     
